### DISCUSSION TITLE
minSdk

### DISCUSSION THREAD

**Discussion by Pingou** [hyochan/react-native-iap#2209] [Q&A] [answered]
As of now, according to https://react-native-iap.dooboolab.com/docs/get-started, the minimum supported Android version is API 24.
Is there any reason it cannot be lower?

**Comment by andresesfm** [comment, marked as answer]
I don't think so, I think the docs are meant to be a guide but as far as I know, billing is supported all the way back to API 8. However, React Native's min is API 21. So I think the min would be 21. I would suggest trying it and posting back if any issues
